src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -49,20 +57,18 @@ public void parseCommand_add() throws Exception {
     }
 
     @Test
-    public void parseCommand_clear() throws Exception {
-        assertTrue(parser.parseCommand(ClearCommand.COMMAND_WORD) instanceof ClearCommand);
-        assertTrue(parser.parseCommand(ClearCommand.COMMAND_WORD + " 3") instanceof ClearCommand);
-    }
+    public void parseCommand_addAppointment() throws Exception {
+        LocalDate date = LocalDate.of(2024, 11, 1);
+        LocalTime startTime = LocalTime.of(9, 0);
+        LocalTime endTime = LocalTime.of(10, 0);
 
+        String input = AddAppointmentCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased() + " "
+                + PREFIX_DATE + date + " " + PREFIX_FROM + startTime + " " + PREFIX_TO + endTime;
 
-    @Test
-    public void parseCommand_delete() throws Exception {
-        DeleteCommand command = (DeleteCommand) parser.parseCommand(
-                DeleteCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased());
-        assertEquals(new DeleteCommand(INDEX_FIRST_MULTIPLE), command);
+        AddAppointmentCommand command = (AddAppointmentCommand) parser.parseCommand(input);
```

Comment:
Is the type cast necessary here? The `assertEquals` test here should still work without it due to polymorphism.

Reply:
Since the existing tests (from AB3) uses type casting, I followed them for consistency. Although I agree that it may not be necessary.